crates/diman_unit_system/src/codegen/float_methods.rs

```diff
@@ -38,26 +51,48 @@ impl Defs {
             self.dimensionless_float_method_for_all_float_types(&quote! { ln }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { log10 }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { exp }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { exp2 }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { ceil }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { floor }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { sin }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { cos }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { tan }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { asin }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { acos }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { atan }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { sinh }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { cosh }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { tanh }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { asinh }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { acosh }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { atanh }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
             self.dimensionless_float_method_for_all_float_types(&quote! { exp_m1 }),
+            #[cfg(any(feature = "std", feature = "num-traits-libm"))]
```

Comment:
I'd prefer it if you could extract these into a single method like
```
#[cfg(any(feature = "std", feature = "num-traits-libm"))]
self.dimensionless_float_methods();
```

Reply:
Done.